contact.php

```diff
@@ -244,14 +244,15 @@
 			);
 			$db->insert_query("maillogs", $log_entry);
 		}
-
-		if($mybb->usergroup['emailfloodtime'] > 0 || (isset($sent_count) && $sent_count + 1 >= $mybb->usergroup['maxemails']))
```

Comment:
Is this line intended to be lost? I don’t see how it relates to the ticket...

Reply:
I hadn't even noticed that somehow.

Reply:
Yes it is intended. That like was for specifically making a decision whether we can / should return to contact.php, on the contrary of the current scenario as we not at all wanna return on a successful submission.

Reply:
Wait wait, these conditions need to be placed somewhere, I need to revise it.

Reply:
Well, I have rechecked it further. Email flood time as well as Sent limit is taken care in the beginning already.
This check is redundant and not applicable as we are not redirecting to `contact.php` further.
I believe what I have done here is correct. @JordanMussi @euantorano